docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -30,26 +30,27 @@ update delay:
       --replicas 3 \
       --name redis \
       --update-delay 10s \
-      --update-parallelism 1 \
       redis:3.0.6
 
     0u6a4s31ybk7yw2wyvtikmu50
     ```
 
     You configure the rolling update policy at service deployment time.
 
-    The `--update-parallelism` flag configures the number of service tasks that
-    the scheduler can update simultaneously. When updates to individual tasks
-    return a state of `RUNNING` or `FAILED`, the scheduler schedules another
-    task to update until all tasks are updated.
-
     The `--update-delay` flag configures the time delay between updates to a
     service task or sets of tasks.
 
     You can describe the time `T` as a combination of the number of seconds
     `Ts`, minutes `Tm`, or hours `Th`. So `10m30s` indicates a 10 minute 30
     second delay.
 
+    By default the scheduler updates 1 task at a time. You can pass the
+    `--update-parallelism` flag to configure the maximum number of service tasks
+    that the scheduler can update simultaneously.
+
+    When an update to an individual task returns a state of `RUNNING` or
+    `FAILED`, the scheduler schedules another task to update until all tasks are
+    updated.
```

Comment:
This will become out of date once #24943 is merged. In fact, it's already out of date, because the version of swarmkit that's not been vendored defaults to pausing the update after a failure, and #24943 merely adds the flag to override this default.